### PR TITLE
OSDOCS-7602: Documented the 4.12.31 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-7602](https://issues.redhat.com/browse/OSDOCS-7602)

Version(s):
4.12

Link to docs preview:
[4.12.31 notes](https://63977--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes#ocp-4-12-31)

QE review:
- [x] QE approval is not required for z-stream updates.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -4001,4 +4001,24 @@ $ oc adm release info 4.12.30 --pullspecs
...
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+
+[id="ocp-4-12-31"]
+=== RHBA-2023:4756 - {product-title} 4.12.31 bug fix update
+
+Issued: 2023-08-31
+
+{product-title} release 4.12.31, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4756[RHBA-2023:4756] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4759[RHBA-2023:4759] advisory.
```
Comment (nalhadef) [inline release_notes/ocp-4-12-release-notes.adoc:4012]: Slight edit for readability cutting down on the number of word.

`{product-title} release 4.12.31 includes security updates. The list of bug fixes included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4756[RHBA-2023:4756] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4759[RHBA-2023:4759] advisory.`

Is the last sentence necessary? It seems to repeat what's in the previous sentence.

**Comment by nalhadef** [comment]
Just one suggestion, Darragh. Otherwise, /lgtm

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-12-release-notes.adoc:4012]: @nalhadef this is stock text for the RNs so if this is edited they would have to edit all entries. Also, to answer your question, yes the the last sentence is necessary as it is the link for the RPMs. The other link is for the images. Hope that helps!

Comment (opayne1) [inline release_notes/ocp-4-12-release-notes.adoc:4012]:
```suggestion
{product-title} release 4.12.31  is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4756[RHBA-2023:4756] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4759[RHBA-2023:4759] advisory.
```

**Review by opayne1 — COMMENTED** [review]
I have one nit @dfitzmau. Let me know when you have it and I can merge!

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-12-release-notes.adoc]: Well spotted, @opayne1 . I need to pay more attention when it comes to bug fix only advisories. Updated.

Comment (dfitzmau) [reply release_notes/ocp-4-12-release-notes.adoc]: Oh, sorry. I missed this thread. Thanks for answering, @opayne1 .

**Pull request merged**